app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PostsController < ApplicationController
+  private
+  def create
+    @Posts = Post.create!(post: params[:post])
+    # grab the user input for params
+    # create a new post from that input
+    # redirect back to the index
+    redirect_to :back
+  end
+  def index
+    @posts = Post.all
+    render :index
```

Comment:
You don't need to indicate that `:index` needs to be rendered. Rails takes care of this automatically with RESTful routes, so long as you set up your routes properly.